packages/nextjs/src/sdk.ts

```diff
@@ -1,2 +1,24 @@
-// eslint-disable-next-line no-console
-console.log('Sentaurs greet NextJS!');
+import { init as browserInit, SDK_VERSION } from '@sentry/browser';
+
+import { NextjsOptions } from './options';
+
+/**
+ * The Sentry NextJS SDK Client.
+ *
+ * TODO: docs, examples...
+ *
+ */
+export function init(options: NextjsOptions): void {
+  options._metadata = options._metadata || {};
+  options._metadata.sdk = {
+    name: 'sentry.javascript.nextjs',
+    packages: [
+      {
+        name: '', // TODO
```

Comment:
```suggestion
        name: '@sentry/nextjs',
```

Reply:
here we should also add

`@sentry/browser` or `@sentry/node` depending on where this was called